### ISSUE TITLE
bug: Fix models uuid documentation

### ISSUE THREAD

**Issue by armondressler** [gobuffalo/buffalo#2372]
#### Description

TLDR: replace "github.com/gobuffalo/uuid" with "github.com/gofrs/uuid" in the docs

This requires a a small change in the documentation. I couldn't find out how to create a PR for the documentation and the link to the CONTRIBUTING document used in https://github.com/gobuffalo/buffalo seems to be broken.

The docs in https://gobuffalo.io/documentation/database/models/ refer to an import of a now archived uuid package:
`"github.com/gobuffalo/uuid"`

This issue has already been discussed in https://github.com/gobuffalo/buffalo/issues/2256#issuecomment-1120420743 . Running `buffalo fix` changes the faulty import to `"github.com/gofrs/uuid"`

### Additional Context

Details

<details>

```
Paste the output of `buffalo info` here!
```

</details>

**Comment by sio4** [comment]
Thank you for reporting that issue!

https://github.com/gobuffalo/docs is the repository for the documentation. Could you please open a PR for the update? I will take care of the CONTRIBUTION file.
